src/test/components/TimelineMarkers.test.js

```diff
@@ -188,17 +197,31 @@ describe('TimelineMarkers', function() {
     delete window.devicePixelRatio;
   });
 
+  it('renders the first marker tooltip when hovered', () => {
+    const { tryToGetMarkerTooltip, mouseOver } = setupWithMarkers(
+      { rangeStart: 0, rangeEnd: 10 },
+      [
+        ['DOMEvent', 0, 3],
+        ['Navigation', 6, 10],
+      ]
+    );
+
+    // Tooltip should not be visible yet.
+    expect(tryToGetMarkerTooltip()).toBeFalsy();
+
+    // The "DOMEvent" marker is drawn from 0,0 to 5,60.
+    mouseOver({ x: 30, y: 2 });
+
+    // It should be visible after hovering the first marker.
+    expect(tryToGetMarkerTooltip()).toBeTruthy();
+  });
+
   describe('displays context menus', () => {
     beforeEach(() => {
       // Always use fake timers when dealing with context menus.
       jest.useFakeTimers();
-      // We will be hovering over element with a tooltip. It requires root overlay
-      // element to be present in DOM
-      addRootOverlayElement();
     });
 
-    afterEach(removeRootOverlayElement);
-
```

Comment:
Since they are moved to the parent describe block, they can be removed from there.